sherpa/sim/tests/test_asymmetric.py

```diff
@@ -182,3 +183,15 @@ def test_ui(self, tol=1.0e-3):
                                   sample['p1.gamma'])
         self.assertEqualWithinTol(self._resample_bench_10['p1.ampl'],
                                   sample['p1.ampl'])
+
+    def test_warning(self):
```

Comment:
@dtnguyen2 - is there any reason you didn't do the fit using the ui layer? Not that it makes a difference here,but it looks a bit odd. That is

    ui.load_ascii_with_errors(1, self.gro_fname)
    ui.set_model(ui.powlaw1d.pl)
    ui.fit()
    with warnings.bla as w:
        more-blah

Reply:
no real reason, other then laziness by cut and paste the code from the original bug report.  Want me to use the ui?

Reply:
No - I don't think so. It just looks a bit odd but I don't think it's a major issue (unless there was something that needed it done this way, but there isn't. So it isn't).